stable/jenkins/templates/config.yaml

```diff
@@ -66,17 +67,29 @@ data:
           <serverUrl>https://kubernetes.default</serverUrl>
           <skipTlsVerify>false</skipTlsVerify>
           <namespace>{{ template "jenkins.master.slaveKubernetesNamespace" . }}</namespace>
-{{- if .Values.master.slaveKubernetesNamespace }}
-          <jenkinsUrl>http://{{ template "jenkins.fullname" . }}.{{ template "jenkins.namespace" . }}:{{.Values.master.servicePort}}{{ default "" .Values.master.jenkinsUriPrefix }}</jenkinsUrl>
-          <jenkinsTunnel>{{ template "jenkins.fullname" . }}-agent.{{ template "jenkins.namespace" . }}:{{ .Values.master.slaveListenerPort }}</jenkinsTunnel>
+{{- if .Values.master.slaveJenkinsUrl }}
+          <jenkinsUrl>{{ .Values.master.slaveJenkinsUrl }}</jenkinsUrl>
```

Comment:
please also add `master.slaveJenkinsUrl` to the JCasC [here](https://github.com/helm/charts/blob/master/stable/jenkins/templates/_helpers.tpl#L93)